e2e/client/playwright/article-send-to.spec.ts

```diff
@@ -0,0 +1,71 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {restoreDatabaseSnapshot, s} from './utils';
+import {TreeSelectDriver} from './utils/tree-select-driver';
+
+test.describe('sending an article', async () => {
+    test('sending an article to another desk', async ({page}) => {
+        const monitoring = new Monitoring(page);
+
+        await restoreDatabaseSnapshot();
+        await page.goto('/#/workspace/monitoring');
+        await monitoring.selectDeskOrWorkspace('Sports');
+
+        await monitoring.executeActionOnMonitoringItem(
+            page.locator(s('article-item=story 2')),
```

Comment:
Update this selector to include current monitoring group. e.g.
`s('monitoring-group=Sports / Working Stage', 'article-item=story 2')`